### ISSUE TITLE
When you go to build_rds() for all the models in 2021

### ISSUE THREAD

**Issue by cgrandin** [pacific-hake/hake-assessment#719]
I updated the section in the README: **How to create the RDS files required for the document to build** for this year clarifying the easiest way to do it.

I also added the RDS file from last year to last year's base model folder because the new code won't build it. It's here: 
https://drive.google.com/file/d/1EX52Q75G_KrQkp-oNexymQQV9u9tTIc5/view?usp=sharing

**Comment by andrew-edwards** [comment]
Thumbs up here when you've downloaded it, then we close.

It's 177Mb (!). That's okay? Seems a bit big....

**Comment by cgrandin** [comment]
It’s last year’s base with everything (forecasting, extra-Mcmc etc) which is why it’s so large

**Comment by andrew-edwards** [comment]
Regarding `build_rds()`, it built all the .rds files for me except one. Trying it again independently gives:
```All built except one (tried it again indepenently)
> build_rds("2021.00.16_h_fix_high")
Creating a new RDS file in C:/andy18/github/hake-assessment/models/2021.00.16_h_fix_high

Error in data.frame(..., check.names = FALSE) : 
  arguments imply differing number of rows: 8000, 7999
> 
```

**Comment by kellijohnson-NOAA** [comment]
I got the same error, it traces back to r4ss::SSgetMCMC. I will look into it.

**Comment by kellijohnson-NOAA** [comment]
This may need to be moved to an adnuts, admb, or r4ss issue, but as of right now `r4ss::SSgetMCMC` blindly combines posteriors and derived posteriors together using `cbind()` when both files have a column called `Iter` (which is the mceval_counter declared in the SS_objfunc.tpl) that could be used to match the two data frames. 
* 2021.00.16_h_fix_high/posteriors.sso is missing 4831
* 2021.00.16_h_fix_high/derived_posteriors.sso is missing 4826 and 4827
So, there are a few issues here,
- [ ] data frames should be matched using Iter or at least checked to see if they have the same number of rows
- [ ] why are the derived posteriors failing for some iterations
- [ ] why did the posterior fail for one
- [ ] is there something going on here with the parallel routine that led to the inconsistency?

**Comment by kellijohnson-NOAA** [comment]
I submitted a stock-synthesis question and reran the model on my machine, the behaviour did not replicate itself in my run. I got 8000 samples per file. I asked Cole, he isn't sure what is going on and hasn't come across it before. Jason Cope (runs mcmc with SS frequently) has experienced weird thing when trying to run SS in parallel. He hasn't investigated them further, just deletes the files and runs again.
